apps/calendar/js/views/advanced_settings.js

```diff
@@ -145,117 +119,60 @@ AdvancedSettings.prototype = {
     router.show(event.target.getAttribute('href'));
   },
 
-  _addAccount: function(id, model) {
-    if (!this._displayAccount(model)) {
-      return;
-    }
-
-    var idx = this.accountList.children.length;
-    var item = template.account.render(this._formatModel(model));
-    this.accountList.insertAdjacentHTML('beforeend', item);
-
-    if (model.error) {
-      this.accountList.children[idx].classList.add('error');
-    }
-  },
-
-  _updateAccount: function(id, model) {
-    var elementId = this.idForModel(ACCOUNT_PREFIX, id);
-    var el = document.getElementById(elementId);
-    if (!el) {
-      return console.error(
-        'trying to update account that was not rendered',
-        id,
-        elementId
-      );
-    }
-
-    if (el.classList.contains('error') && !model.error) {
-      el.classList.remove('error');
-    }
+  render: co.wrap(function *() {
+    this.header.runFontFitSoon();
 
-    if (model.error) {
-      el.classList.add('error');
-    }
-  },
+    // the accounts and syncFrequency are already handled by the _initEvents
+    var bridge = core.bridge;
+    var data = yield {
+      standardAlarmDefault: bridge.getSetting('standardAlarmDefault'),
+      alldayAlarmDefault: bridge.getSetting('alldayAlarmDefault')
+    };
 
-  _removeAccount: function(id) {
-    var el = document.getElementById(this.idForModel(ACCOUNT_PREFIX, id));
+    this._renderAlarmDefault('standard', data.standardAlarmDefault);
+    this._renderAlarmDefault('allday', data.alldayAlarmDefault);
+  }),
 
-    if (el) {
-      /** @type {Node} */
-      var parentNode = el.parentNode;
-      parentNode.removeChild(el);
-    }
+  _renderSyncFrequency: function(value) {
+    this.syncFrequency.value = String(value);
   },
 
-  render: function() {
-    var self = this;
-    var pending = 4;
+  _renderAccounts: function(accounts) {
+    this.accountList.innerHTML = '';
 
-    function next() {
-      if (!--pending && self.onrender) {
-        self.onrender();
-      }
-    }
+    accounts
+    .filter(o => o.provider.hasAccountSettings)
```

Comment:
will rename to `data` to match the `forEach`.. I probably had it on a single line before and used `o` so it could fit (o == object)